deploy/helm/templates/nmstate.io_v1_nmstate_cr.yaml

```diff
@@ -0,0 +1 @@
+../../../build/_output/manifests/nmstate.io_v1_nmstate_cr.yaml
```

Comment:
Those are all dangling symlinks, i.e. when you just `git clone` they will not point to existing files, not even to the directory structure.

I would rather keep `deploy/` in the repository clean and have something like

```
mkdir -p deploy/helm/templates
cp ../../../build/_output/manifests/* deploy/helm/templates/
```
inside the pipeline, so after `make manifests`.

Reply:
@supu2 can you address this comments, if not we will close this PR.